docs/gettingstarted.rst

```diff
@@ -242,13 +242,62 @@ environment, put it in ``my.env``.
 General architecture
 ====================
 
+.. graphviz::
+
+   digraph G {
+      rankdir=LR;
+
+      client [shape=box3d, label="firefox"];
+
+      subgraph antpig {
+         rank=same;
+
+         antenna [shape=rect, label="antenna"];
+         pigeon [shape=cds, label="pigeon"];
+      }
+
+      subgraph stores {
```

Comment:
Grouping all the stores together makes the lines to a from the processor a little difficult to read, could we split this into multiple groups?